kubelet/datadog_checks/kubelet/__about__.py

```diff
@@ -2,4 +2,4 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 
-__version__ = "3.4.0"
```

Comment:
you shouldn't change this manually, it will be bumped automatically when a new version of the integration is relased

Reply:
Ah yeah this tripped the version check in the build. Thanks!